app/models/competition.rb

```diff
@@ -2284,6 +2284,9 @@ def to_form_data
         "forceComment" => force_comment_in_registration,
       },
       "eventRestrictions" => {
+        "newcomersAllowed" => {
+          "enabled" => newcomers_allowed?,
+          "reason" => newcomers_allowed,
```

Comment:
You're missing a closing bracket here :wink:

Reply:
Changed